app/Http/Transformers/ReportbackTransformer.php

```diff
@@ -36,9 +38,20 @@ public function transform(Post $post)
                 'flagged' => 'false',
             ],
             'kudos' => [
```

Comment:
I think this could be an embedded response that uses the already defined `KudoTransformer` so you don't have to set it up twice. For example, in the [SignupTransformer](https://github.com/DoSomething/rogue/blob/master/app/Http/Transformers/SignupTransformer.php), by default we `includePosts()` which allows us to use the `PostTransformer` in the response.